app/views/works/main.html.erb

```diff
@@ -0,0 +1,65 @@
+<main class="row">
+  <div>
+    <h3>
+      <span class="media-spotlight-header"></span>
+      Media Spotlight:
+      <% if @works.first != nil %>
+      <%= link_to("#{Work.most_popular.title}", work_path(Work.most_popular)) %>
+      by <%= Work.most_popular.creator %>
+      <% else %>
+      <%= link_to("Add some works!", new_work_path) %>
+      <% end %>
+    </h3>
+    <p><%= "#{Work.most_popular.votes.count} votes" %>
```

Comment:
You don't really need to do string interpolation in your views, you should be able to put the code in the code tags and then the remaining text outside: `<%= Work.most_popular.votes.count %> votes`